client/src/components/CreatePost/CustomModal.js

```diff
@@ -42,7 +63,19 @@ const CustomModal = ({ title, content, onClose, visible, closable }) => {
       transparent
       animationType="slide-up"
     >
+      {searchBar ? <SearchInput
+        onChange={onSearch}
+      /> :
+      null }
       {content}
+      {closeButton ?
+        <SubmitButton
+        style={buttonStyles}
+        onClick={selectCloseButton}
+        >
+        {closeButtonName}
+        </SubmitButton> : null
+      }
```

Comment:
So is this using `CustomModal` from `CreatePost` for creating organization?

Reply:
Yea. Alex used it but I didn't see any reason to remove it. Any issues?